static/src/javascripts/projects/common/modules/identity/cookierefresh.js

```diff
@@ -1,28 +1,30 @@
 import {
-    isUserLoggedIn,
-    getUserFromApiWithRefreshedCookie,
+	isUserLoggedIn,
+	getUserFromApiWithRefreshedCookie,
+	refreshOktaSession,
 } from 'common/modules/identity/api';
-import { storage } from '@guardian/libs'
+import { storage } from '@guardian/libs';
 
-const shouldRefreshCookie = (
-    lastRefresh,
-    currentTime
-) => {
-    const days30k = 1000 * 86400 * 30; // (as seconds)
-    return !lastRefresh || currentTime > parseInt(lastRefresh, 10) + days30k;
+const shouldRefreshCookie = (lastRefresh, currentTime) => {
+	const days30InSeconds = 1000 * 86400 * 30; // (as seconds)
```

Comment:
**If** we use other quantities of days it might be good to have a const of `dayInSeconds` and multiply that where used

Reply:
This is actually millis now that I look at it again so I'll make the logic here a bit clearer

Reply:
You can also use underscores, now, to make large values easier to parse, like so: `86_400`.